apps/nova/src/pages/UpcomingLessons.tsx

```diff
@@ -37,6 +96,36 @@ const UpcomingLessons: React.FC = () => {
         hasMore={lessons.query.hasNextPage && !lessons.query.isPending}
         refetch={lessons.query.refetch}
       />
+      <RatingDialog
+        open={rateDialogInfo.enabled}
+        loading={tutorQuery.isLoading || ratingQuery.isLoading}
+        studentId={user?.id || -1}
+        studentName={user?.name || ""}
+        tutorName={tutorQuery.data?.name || ""}
+        imageUrl={tutorQuery.data?.image || ""}
+        feedback={rateDialogInfo.rateFeedback || ""}
+        rating={rateDialogInfo.rateValue || 0}
```

Comment:
No need to pass empty or default values. Only render the dialog if you have all the required information. 

```ts
{tutorQuery.data && user ? <RatingDialog /> : null}
```